### PR TITLE
Add some lines of code

### PR DESCRIPTION
**Pull request by riko105**

### CODE REVIEW

File: hello.swift [hunk 1]
```diff
@@ -3,3 +3,13 @@ import Foundation
 
 print("Hello World!")
 
+let step = 20
+
+// sorry Michael, don't know exelent Swift syntax yet
```
Comment (andrewwoz) [inline hello.swift]: It's not a problem

Comment (riko105) [inline hello.swift]: ok, that's sounds good!

**Comment by andrewwoz** [comment]
You must write detailed description before making pull request.

**Comment by riko105** [comment]
Ok

**Comment by andrewwoz** [comment]
Are you sure this code is necessary?

**Comment by riko105** [comment]
yes, cause this is my code)))

**Comment by riko105** [comment]
That's better)

**Comment by andrewwoz** [comment]
I have merged with fast-forward

**Pull request merged**